networktest.go

```diff
@@ -185,6 +185,7 @@ func (l *lightningNode) genArgs() []string {
 	args = append(args, fmt.Sprintf("--configfile=%v", l.cfg.DataDir))
 	args = append(args, fmt.Sprintf("--adminmacaroonpath=%v", l.cfg.AdminMacPath))
 	args = append(args, fmt.Sprintf("--readonlymacaroonpath=%v", l.cfg.ReadMacPath))
+	args = append(args, fmt.Sprintf("--externalip=%v", l.cfg.ExternalIPs))
```

Comment:
This isn't necessary, as the `ExternalIPs` parameter is never set (in normal test operation). Also, assigning the `--externalip` to an empty string as done here, may trigger some odd behavior in the tests due to the way resolution is currently attempted. As an example, here's the result of me setting `--externalip` twice locally (once with `localhost` and the other with no parameter):
```
{
    "nodes": [
        {
            "last_update": 1506398714,
            "pub_key": "02a196c6106ba977647119414dd8da019a708cdeed2b8ff7e1c691b0aa329643ba",
            "alias": "02a196c6106ba9776471\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000",
            "addresses": [
                {
                    "network": "tcp",
                    "addr": "127.0.0.1:9735"
                },
                {
                    "network": "tcp",
                    "addr": "[107:1:59c9:d1fa::]:9735"
                }
            ]
        }
    ],
    "edges": [
    ]
}

```

I see you added this in order to be able to exercise this new behavior within the integration tests. But I'm not convinced it's really necessary as this is such a trivial feature and the added tests will slow down the execution of all the integration tests a bit (until we move to parallelizing all test execution).

Reply:
I agree that the test is superfluous. My latest commit reverts the integration test. Let me know if any other changes are needed.